fsspec/registry.py

```diff
@@ -80,9 +51,6 @@ def register_implementation(name, cls, clobber=True, errtxt=None):
         _registry[name] = cls
 
 
-registry = ReadOnlyRegistry(_registry)
-default = "file"
```

Comment:
This was not used anywhere?

Reply:
I moved it to the top where it was more obvious. Is that OK?

Reply:
Oh yes, simply didn't notice where it had gone.